src/agents/supervisor.py

```
@@ -0,0 +1,109 @@
+"""Supervisor agent for routing and orchestration."""
+
+import logging
+from typing import Dict, Any
+from langchain_core.messages import HumanMessage
```

Ruff (0.12.2)

`langchain_core.messages.HumanMessage` imported but unused

Remove unused import: `langchain_core.messages.HumanMessage`

(F401)